api/sso/sso_api/endpoints/password_change.py

```diff
@@ -1,36 +1,34 @@
-# Mycroft Server - Backend
-# Copyright (C) 2019 Mycroft AI Inc
-# SPDX-License-Identifier: 	AGPL-3.0-or-later
+#  Mycroft Server - Backend
+#  Copyright (c) 2022 Mycroft AI Inc
+#  SPDX-License-Identifier: 	AGPL-3.0-or-later
+#  #
+#  This file is part of the Mycroft Server.
+#  #
+#  The Mycroft Server is free software: you can redistribute it and/or
+#  modify it under the terms of the GNU Affero General Public License as
+#  published by the Free Software Foundation, either version 3 of the
+#  License, or (at your option) any later version.
+#  #
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+#  GNU Affero General Public License for more details.
+#  #
+#  You should have received a copy of the GNU Affero General Public License
+#  along with this program. If not, see <https://www.gnu.org/licenses/>.
 #
-# This file is part of the Mycroft Server.
-#
-# The Mycroft Server is free software: you can redistribute it and/or
-# modify it under the terms of the GNU Affero General Public License as
-# published by the Free Software Foundation, either version 3 of the
-# License, or (at your option) any later version.
-#
-# This program is distributed in the hope that it will be useful,
-# but WITHOUT ANY WARRANTY; without even the implied warranty of
-# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
-# GNU Affero General Public License for more details.
-#
-# You should have received a copy of the GNU Affero General Public License
-# along with this program. If not, see <https://www.gnu.org/licenses/>.
+"""Defines the password change endpoint for the account API."""
```

Comment:
Is this meant to be here? 

There is an endpoint in both the account and sso API's?

Reply:
Yes.  There is a base class in the selene package.  That base class is overridden in different ways in the single sign on and account APIs since in one you are authenticated and in the other you are not.